### ISSUE TITLE
cerebellar tonsillar ectopia

### ISSUE THREAD

**Issue by RiyaUthup** [obophenotype/human-phenotype-ontology#10837]
**Preferred term label:**cerebellar tonsillar ectopia

**Synonyms**

**Definition (free text, please give PubMed ID)**38833623

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@RiyaUthup is this a synonym for [Chiari malformation HP:0002308](https://hpo.jax.org/browse/term/HP:0002308)? If not please explain the difference.

**Comment by RiyaUthup** [comment]
It’s the same. Could you please add the synonym?

**Comment by pnrobinson** [comment]
@RiyaUthup added as synonym, thanks!
